docs/guides/tutorials/prefarm-audit.md

```diff
@@ -0,0 +1,121 @@
+---
+slug: /guides/custody-tool-description
+title: Custody Tool Description
+---
+
+Chia Network Inc's prefarm is being secured by a complex set of custodial rules. This document will describe how to use the custody tool to audit the prefarm configuration. A moderate level of technical proficiency is probably needed to understand the details. For a high-level overview of the prefarm custody wallets, see our [blog post](https://www.chia.net/2022/10/29/a-new-home-for-the-prefarm/).
```

Comment:
Suggestion: Use active voice wherever possible. 
Exs.: ...prefarm is secured by...
This document describes...